docs/source/guides/quickstart.md

```diff
@@ -1,48 +1,149 @@
 # NuCypher Quickstart
 
-## Ursula
 
-### Install NuCypher
+## A Note about Side-Channels
 
+The NuCypher network does not store your application's data; Instead - it manages access *to* your
+application's data. Management of encrypted secrets and public keys is highly application-specific - 
+In general, nucypher needs to be integrated with a storage or transport layer to function properly.
+Along with the transport of ciphertext, the application also needs to include 
+facilities for several other pieces of cryptographic material, specifically, channels
+for Alice and Bob to discover each-others public keys, as well as a way to provide policy public keys 
+to Bob and Enrico.
+ 
+##### Side Channel Data:
+  - Secrets:
+    - Message Kits (Ciphertext)
+    - Labels
+  - Identities:
+    - Alice Verifying Key
+    - Bob Encrypting Key
+    - Bob Verifying Key
+  - Policies:
+    - Policy Encrypting Key 
+
+
+## Connecting to the NuCypher Network
+
+### Running an Ethereum Node
+
+Operation of nucypher cryptological characters `Alice`, `Bob`, and `Ursula` require
+a running ethereum node and wallet (We recommend Geth).
+
+To run a Goerli connected Geth node:
 ```bash
-$ pip3 install -U nucypher
+$ geth --goerl
 ```
 
-### Run a Federated-Only Development Ursula
+### Provider URI
 
-```bash
-$ nucypher ursula run --dev --federated-only
+Nucypher uses the node's IPC-File to communicate, specified by `provider_uri`.
+By default in ubuntu, the path is `~/.ethereum/goerli/geth.ipc` - This path
+will also be logged to the geth-running console on startup. 
+
+### Ursula: Untrusted Re-Encryption Proxies
+
+When initializing an `Alice`, `Bob`, or `Ursula`, an initial "Stranger-`Ursula`" is needed to perform 
+the role of a `Teacher`, or "seednode".  The `known_nodes` will inform your character of all of the nodes
+they know about network-wide, then kick-off the automated node-discovery loop.
+
+Stranger `Ursula`s can be created by invoking the `from_seed_and_stake_info` method, then a `list` of `known_nodes`
```

Comment:
I would swap this around so you first instruct users how to invoke `Stranger-Ursula` then explain what the `known-nodes` do.

Reply:
:heavy_check_mark: